social_protection/models.py

```diff
@@ -20,4 +28,14 @@ class BenefitPlan(core_models.HistoryBusinessModel):
 class Beneficiary(core_models.HistoryBusinessModel):
     individual = models.ForeignKey(Individual, models.DO_NOTHING, null=False)
     benefit_plan = models.ForeignKey(BenefitPlan, models.DO_NOTHING, null=False)
-    status = models.CharField(max_length=100, null=False)
+    status = models.CharField(max_length=100, choices=StatusChoices.choices, null=False)
+
+
+class Group(core_models.HistoryBusinessModel):
+    benefit_plan = models.ForeignKey(BenefitPlan, models.DO_NOTHING, null=False)
+    status = models.CharField(max_length=100, choices=StatusChoices.choices, null=False)
+
+
+class GroupIndividual(core_models.HistoryBusinessModel):
```

Comment:
I think this should be placed in 'Individual' module.

Reply:
yes or group module